tensorboard/backend/application_test.py

```diff
@@ -339,5 +339,23 @@ def testExceptions(self):
       application.TensorBoardWSGIApp(logdir, plugins, multiplexer, 0)
 
 
+class DbTest(tf.test.TestCase):
+
+  def testSqliteDb(self):
+    self.addCleanup(shutil.rmtree, self.get_temp_dir())
```

Comment:
[You don't need to do this.](https://github.com/tensorflow/tensorflow/blob/38e0922d1e2dcd572379af4496f878492e9f689a/tensorflow/python/platform/test.py#L73-L76)

(Personally, I prefer not removing the temporary directories, so that I can inspect them. This has proven useful. Your call.)